practical-07/src/components/NavBar.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <nav class="fixed-top">
+    <div class="nav-bar">
+    <div>
+      <router-link to="/" class="home-btn nav-link">Home</router-link>
+    </div>
+    <div class="nav-heading" v-if="!['LoginForm', 'RegisterForm'].includes($route.name)">
+      <span>{{ heading }}</span>
```

Comment:
Codacy has a fix for the issue: [Insert `··`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
        <span>{{ heading }}</span>
```